SeleniumPracticalExercise/PageObjects/AddEmployeePage.cs

```diff
@@ -0,0 +1,41 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using SeleniumPracticalExercise.PageObjects.Common;
+
+namespace SeleniumPracticalExercise.PageObjects
+{
+    public class AddEmployeePage : BasePageLocal
+    {
+        private readonly By _employeeIdLocator = By.CssSelector("div.oxd-grid-2 input");
+        private readonly By _firstNameFieldLocator = By.CssSelector("input[name='firstName']");
+        private readonly By _lastNameFieldLocator = By.CssSelector("input[name='lastName']");
+        private readonly By _saveButtonLocator = By.CssSelector("button[type='submit']");     
+        
+        public AddEmployeePage(IWebDriver driver) : base(driver)
+        {
+        }
+
+        /// <summary>
+        /// Enter Firstname & Lastname
+        /// then Click Save
+        /// did stale so we can wait for element to save. 
+        /// </summary>
+        public void SetEnterNames(string firstname, string lastname)
```

Comment:
You are missing XML doc parameters. Add those here and check all the other docs for missing params.

Reply:
went through all page objecs I created and re-did the XML documentation